packages/deployer/subtasks/clear-deployments.js

```diff
@@ -1,6 +1,5 @@
 const del = require('del');
 const { subtask } = require('hardhat/config');
-const { TASK_CLEAN } = require('hardhat/builtin-tasks/task-names');
```

Comment:
Also this diff should go once the feature branch is updated to main.

Reply:
fixed